gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagerMetrics.java

```diff
@@ -76,8 +76,9 @@ public class DagManagerMetrics {
   private final Map<String, ContextAwareMeter> executorSlaExceededMeters = Maps.newConcurrentMap();
   private final Map<String, ContextAwareMeter> executorJobSentMeters = Maps.newConcurrentMap();
 
-  // Metrics for unexpected flow handling failures
+  // Metric for unexpected flow handling failures
   private ContextAwareCounter failedLaunchEventsOnActivationCount;
+  private ContextAwareCounter successfulLaunchEventsOnActivationCount;
```

Comment:
now these are for flow handling outcomes (although they're both only for startup, so not in the general sense)